yxi/axi-calyx/axi-combined-calyx.futil

```diff
@@ -902,9 +924,13 @@ component main(
             //read stuff
             par{
                 //init base_addresses
-                invoke base_addr_A0(in = 64'b0)();
-                invoke base_addr_B0(in = 64'b0)();
-                invoke base_addr_Sum0(in = 64'b0)();
+                //TODO: get this from kernel.xml
```

Comment:
Just a high-level notion: in the limit, let's actually *generate* `kernel.xml`!

That is, what currently happens is: we run the Calyx compiler once to produce `kernel.xml`, and then another time to generate the corresponding Verilog code for the AXI interface.

What we could instead do is: we run the Calyx compiler once to produce the YXI file. Then, our new tooling runs (consuming only the YXI file) and produces two things: (1) the Calyx code for the AXI interface, and (2) `kernel.xml`.

(So basically, I'm suggesting that we (YXI) be in charge, rather than the existing `kernel.xml` generator, which we can eventually remove.)